src/Parsers/ASTCreateQuery.h

```diff
@@ -25,6 +25,8 @@ class ASTStorage : public IAST
     IAST * ttl_table = nullptr;
     ASTSetQuery * settings = nullptr;
 
+    ASTPtr comment;
```

Comment:
```suggestion
    IAST * ttl_table = nullptr;
    IAST * comment = nullptr;
    ASTSetQuery * settings = nullptr;
```
Why is it not like this? all other such values are just raw pointers `IAST *`, and this one is no different from them.

Reply:
OK! updated.